### PR TITLE
January 12, 2023 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier, @robertengaxway, @georgiana-caras - Ready for content

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Product plan archival from UI** - From the webUI, the ellipsis menu allow to archive a deprecated plan. In case there are associated subscription, the plan archival is not permitted and the menu is replaced with a message informing there are subscription associated with the plan
+
+#### Bug fixes for Product Foundry
+
+* **Long product name are badly displayed when Activating the product** - Long product name are no more truncated when activating a product version.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:127]: **Long product names are not properly displayed when activating a product** - Long product names are no longer truncated  when activating a product version.

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+* **Permission issue when editing product and asset has read-only access** - The Asset name attached to a product are always visible. In case the user cannot access it, the link will be disable
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:128]: Prior to the fix, an user that had edit rights on a product that contained an asset that was shared with his / her team in read-only mode,  could not see the asset name. After the fix, the asset name is visible. In situations where the user does not have access to the asset, the link will be disabled so they cannot see the details of the asset.

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+* **Switching back and force between paid and free plan in the wizard does not create the plan correctly** - The wizard page navigation is now correctly handled and the plan is created according to the last information on the screen.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:129]: **Switching back and fort between a paid and a free plan in the wizard does not create the plan correctly**

**Review by robertengaxway — APPROVED** [review]
I approve

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Documentation real estate** - The support contact / category / tags and product history are hidden on the product documentation page to allow more space for viewing the documentation.
+
+#### Bug fixes for Marketplace
+
+* **Featured categories are overlapping** - The display of 5 featured category has been enhanced to allow viewing correctly the category name.
+* **Support contact long name no well no visible in marketplace** - Now the support contact name is truncated based on the space available to display it and a tooltip shows its entire value.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:152]: **Long support contact name not properly displayed in the Marketplace** - Long support contact names are not truncated to fit on the screen and a tool tip has been added to show the entire value when user hovers over it.

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+* **Untimely scrolling issue when searching product in the Marketplace home page** - The untimely scroll have been removed and now the search is smoother.
+* **Application details screen does not allow to view the credentials** - When a user wanted to see his credentials, the view credentials button was closing the credentials panel.
+* **The section navigation in documentation is broken when using special characters ('?', '#', '/'...)** - The special characters are now accepted in documentation section and does not break the navigation.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:155]: **The navigation section in the documentation is broken when using special characters ('?', '#', '/'...)**

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+##### Feature updates for Axway API Management Gateway
+
+There are no new features in this release.
+
+##### Bug fixes for Axway API Management Gateway
+
+* **Duplicate API Service Revisions** - There was a fix to not create duplicate API service revisions when restrating the Discovery agent.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:52]: when restarting the Discovery agent.

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+There are no enhancements or bug fixes in this release.
+
+### Asset Catalog
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Asset Catalog
+
+* **Asset archival from UI** - From the webUI Asset Catalog, the ellipsis menu allows an asset to be archived.   All asset releases will be moved to an arhived state as long as there are no assoicated Products consuming the asset release.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:105]: typo **arhived**

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Asset Catalog
+
+* **Asset archival from UI** - From the webUI Asset Catalog, the ellipsis menu allows an asset to be archived.   All asset releases will be moved to an arhived state as long as there are no assoicated Products consuming the asset release.  
+
+#### Bug fixes for Asset Catalog
+
+* **Asset/Product Search Bar UI** - The search bar handles the use of special characters (e.g. #, $, %, etc) without generating an error message.
+* **Asset Creation UI** - A fix was made to the Asset creation UI wizard to without generating an error message when saving the asset.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:110]: I don;t understand what was generating the error.

File: content/en/docs/amplify_relnotes/20230112_amplify.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+* **Asset status Ui** - A fix was made to update the Asset version status and the overall Asset status in a more timely fashion.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230112_amplify.md:111]: UI

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Pull request merged**